3_javascript/javascript1/week1/javascript.js

```diff
@@ -0,0 +1,104 @@
+ //hello... Hello... This is my first experience in Javascript :)
+
+console.log("******    Age-ify (A future age calculator)    ******");
+let myYearOfBirth = 1991;
+let yearFuture = 2027;
+let age = yearFuture - myYearOfBirth;
```

Comment:
Great Job! Just regarding using the `let` keyword.  We usually use it only if we want to re-assign the variable later, but in our case, it's better to use constant `const,` because we want to keep the value the same for the whole time
example of using `let`
```
let name = "peter";

console.log(name); // peter

name = "lucas";

console.log(name); // lucas


```

Reply:
great explanation... love it :)
done!